jobs/async-upload/job/config.py

```diff
@@ -114,9 +114,9 @@ def _load_s3_credentials(path: str | Path, store: S3Config) -> None:
     The path must be a folder, with a number of files that match a typical AWS config, ie - a Secret mounted to a pod with keys:
     - AWS_ACCESS_KEY_ID
     - AWS_SECRET_ACCESS_KEY
-    - AWS_BUCKET
-    - AWS_REGION
-    - AWS_ENDPOINT_URL
+    - AWS_S3_BUCKET or AWS_BUCKET
+    - AWS_DEFAULT_REGION
```

Comment:
thank you @Crazyglue for this update, but I'm a little confused how this is accounted in the below, please.

When a Secret is :

```yaml
AWS_SECRET_ACCESS_KEY:  ...
AWS_ACCESS_KEY_ID:      ...
AWS_DEFAULT_REGION:     ...
AWS_S3_BUCKET:          ...
AWS_S3_ENDPOINT:        ... 
```

how does AWS_DEFAULT_REGION get deserialized into `region` field? 🤔

Reply:
I think that `AWS_S3_ENDPOINT ` has the same problem, I am getting  
```
ValueError: "S3Config" object has no field "s3_endpoint"
```